[Code Quality] Issue in lib/executor/instantiate/elem.cpp

```
/home/runner/work/WasmEdge/WasmEdge/lib/executor/instantiate/elem.cpp:48: error: Null Dereference
  pointer `TabInst` last assigned on line 46 could be null and is dereferenced at line 48, column 14.
  46.         auto *TabInst = getTabInstByIdx(StoreMgr, ElemSeg.getIdx());
  47.         /// Check elements fits.
  48.         if (!TabInst->checkAccessBound(
                   ^
  49.                 Offset, static_cast<uint32_t>(InitVals.size()))) {
  50.           spdlog::error(ErrCode::ElemSegDoesNotFit);
```

--- a/lib/executor/instantiate/elem.cpp
+++ b/lib/executor/instantiate/elem.cpp
@@ -45,6 +45,7 @@ Expect<void> Executor::instantiate(Runtime::StoreManager &StoreMgr,
         /// Table index should be 0. Checked in validation phase.
         auto *TabInst = getTabInstByIdx(StoreMgr, ElemSeg.getIdx());
         /// Check elements fits.
+        assert(TabInst != nullptr);
         if (!TabInst->checkAccessBound(
                 Offset, static_cast<uint32_t>(InitVals.size()))) {
           spdlog::error(ErrCode::ElemSegDoesNotFit);
@@ -77,6 +78,7 @@ Expect<void> Executor::initTable(Runtime::StoreManager &StoreMgr,
   uint32_t Idx = 0;
   for (const auto &ElemSeg : ElemSec.getContent()) {
     auto *ElemInst = getElemInstByIdx(StoreMgr, Idx);
+    assert(ElemInst != nullptr);
     if (ElemSeg.getMode() == AST::ElementSegment::ElemMode::Active) {
       /// Table index is checked in validation phase.
       auto *TabInst = getTabInstByIdx(StoreMgr, ElemSeg.getIdx());

